scripts/core/editor3/components/BaseUnstyledComponent.tsx

```diff
@@ -51,7 +51,8 @@ class BaseUnstyledComponent extends React.Component<any, any> {
         const blockKey = this.getDropBlockKey();
         const link = event.originalEvent.dataTransfer.getData('URL');
 
-        if (canDropMedia(event, this.props.editorProps) && mediaType.includes('application/superdesk')) {
+        if (canDropMedia(event, this.props.editorProps)
+            && (mediaType === 'Files' || mediaType.includes('application/superdesk'))) {
```

Comment:
Move this part `&& (mediaType === 'Files' || mediaType.includes('application/superdesk'))` to the body of `canDropMedia` as well.

Reply:
It's there to differentiate if a file should go as media or embed.